extlinks/organisations/templates/organisations/organisation_charts_include.html

```diff
@@ -139,6 +146,20 @@ <h3>Latest link events</h3>
         },
       },
     });
+
+    // Click all Link Events chart section TESTETETETETETETETET
+    document.getElementById('{{ collection_name }}_linkEvents_button').click();
+    var form_data = {{ form_data|safe }};
+    var collection_id = {{ collection.collection_id|safe }};
+    console.log("Loading information for collection " + collection_id);
```

Comment:
Do we want this console.log to go to main?

Reply:
Deleted! That was a leftover debugging console.log.